tests/integration/sync_test.go

```diff
@@ -7652,3 +7652,162 @@ func Test_Sync_CustomEntities_Fake(t *testing.T) {
 	require.Error(t, err)
 	assert.ErrorContains(t, err, "unknown entity type: fake-entity")
 }
+
+func Test_Sync_Partials_Plugins(t *testing.T) {
+	client, err := getTestClient()
+	require.NoError(t, err)
+
+	ctx := context.Background()
+
+	runWhen(t, "enterprise", ">=3.10.0")
+	dumpConfig := deckDump.Config{}
```

Comment:
No need to create client if Kong version is not as expected.

```suggestion
	runWhen(t, "enterprise", ">=3.10.0")

	client, err := getTestClient()
	require.NoError(t, err)

	ctx := t.Context()

	dumpConfig := deckDump.Config{}
```